arangod/Aql/OptimizerRules.cpp

```diff
@@ -1933,7 +1933,7 @@ void arangodb::aql::specializeCollectRule(Optimizer* opt,
   for (auto const& n : nodes) {
     auto collectNode = ExecutionNode::castTo<CollectNode*>(n);
 
-    if (collectNode->isSpecialized()) {
+    if (collectNode->isFixedMethod()) {
       // already specialized this node
```

Comment:
This comment might now need an adjustment?

Reply:
Addressed